packages/slice-machine/src/legacy/lib/models/common/widgets/Widget.ts

```diff
@@ -23,5 +24,13 @@ export interface Widget<F extends TabField, S extends AnyObjectSchema> {
   prepareInitialValues?: (props: F["config"]) => F["config"];
 }
 
+export type Widget<F extends TabField, S extends AnyObjectSchema> = F extends
```

Comment:
🎉 #nice

I like, it forces slice machine to not forget about the `hintItemName`  for group.
In the future it will be even better because the source of truth will be adapters, but for today I think it's perfect!